### PR TITLE
Fine tuning scripts update

### PR DESCRIPTION
**Pull request by Peilin-Yang**

### CODE REVIEW

File: src/main/python/fine_tuning/xfold.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 import logging
 import json
 
-logging.basicConfig()
```
Comment (lukuang) [inline src/main/python/fine_tuning/xfold.py]: You added `logging.basicConfig()` for other files but remove it from this file. Any particular reason?

Comment (Peilin-Yang) [reply src/main/python/fine_tuning/xfold.py]: No idea why it was removed.
Reverted

File: src/main/resources/fine_tuning/models.yaml [hunk 2]
```diff
@@ -102,24 +122,26 @@ models:
...
+          best_avg: 0.0
+          oracles_per_topic: 0.0
+          2-fold: 0.0
+          5-fold: 0.0
+        ndcg20:
+          best_avg: 0.0
+          oracles_per_topic: 0.0
+          2-fold: 0.0
+          5-fold: 0.0
   rm3:
```
Comment (lukuang) [inline src/main/resources/fine_tuning/models.yaml]: For `rm3` and `bm25+rm3`, `originalQueryWeight` can be `1.0`. I think we should have that. But then all combinations of other `rm3` parameters will be searched for `originalQueryWeight=1.0`. Then the whole program wastes some time on those? Can we get around it such as support having multiple parameter ranges? Not necessary needs to be changed in this PR but I think we probably should avoid that?

File: src/main/resources/fine_tuning/models.yaml [hunk 3]
```diff
@@ -174,30 +204,20 @@ models:
...
-        pace: 0.1
     expected:
       robust04:
         map:
-          best_avg: 0.2532
-          oracles_per_topic: 0.2921
-          2-fold: 0.2253
-          5-fold: 0.2524
+          best_avg: 0.3020
+          oracles_per_topic: 0.4343
```
Comment (lukuang) [inline src/main/resources/fine_tuning/models.yaml]: should be `0.4402`? Please check out the newest version of the paper to updated numbers

File: src/main/resources/fine_tuning/models.yaml [hunk 4]
```diff
@@ -174,30 +204,20 @@ models:
...
+          2-fold: 0.2973
+          5-fold: 0.2956
         P_20:
-          best_avg: 0.3614
-          oracles_per_topic: 0.4426
-          2-fold: 0.3604
-          5-fold: 0.3577
+          best_avg: 0.4012
+          oracles_per_topic: 0.5960
```
Comment (lukuang) [inline src/main/resources/fine_tuning/models.yaml]: `0.6054`

File: src/main/resources/fine_tuning/models.yaml (hunk 3 above)
Comment (Peilin-Yang) [reply src/main/resources/fine_tuning/models.yaml]: I will leave it until we merge your PR? (actually you can update it?)

Comment (lukuang) [reply src/main/resources/fine_tuning/models.yaml]: I guess that's fine too

**Pull request merged**